pyfixest/report/visualize.py

```diff
@@ -436,15 +437,17 @@ def _coefplot_matplotlib(
     flip_coord: Optional[bool] = True,
     labels: Optional[dict] = None,
     ax: Optional[plt.Axes] = None,
+    dodge: float = 0.5,
```

Comment:
hardcoded this for now since I didn't want to pass along many matplotlib-specific parameters from the `iplot` layer downwards. For sufficiently persnickety users (e.g. me), `tidy()` exists.